lib/tasks/grda_warehouse.rake

```diff
@@ -322,6 +322,8 @@ namespace :grda_warehouse do
 
     stats_collector = AppResourceMonitor::CollectStatsJob.new
     AppResourceMonitor::CollectStatsJob.perform_later if stats_collector.should_enqueue?
+
+    BuildTranslationCacheJob.perform_later
```

Comment:
This would occur hourly instead of every 15min. Is that okay? Could also call this job from an after_commit hook on Translation

Reply:
Yeah, this feels completely fine.  It should still update the particular translation after it has been saved, the every 15 minutes was really just a failsafe for if we missed updating the cache when changing a translation somehow.